src/Inter/cps.rs

```
@@ -1016,6 +1051,49 @@ impl Environment {
         }
     }
 
+    pub fn define_enum(&mut self, name: &String, variants: &Vec<String>) -> Result<(), CPSError> {
```

Clippy (1.97.1)

[warning] the `Err`-variant returned from this function is very large

(warning)